.goreleaser.yml

```diff
@@ -11,8 +11,6 @@ builds:
       - arm64
 
 archives:
-  - id: zips
-    name_template: "{{ .ProjectName }}_{{ .Os }}"
```

Comment:
I removed the old one because I was getting that conflict.

I think we could copy these as a separate step if we want. We already do some of that for s3